src/components/MarketMySharesForm/index.js

```diff
@@ -61,7 +56,7 @@ class MarketMySharesForm extends Component {
       const share = marketShares.filter(_share => _share.id === extendedSellId)[0]
       const fullAmount = Decimal(share.balance)
         .div(1e18)
-        .toDP(4, 1)
+        .toDP(2, 1)
```

Comment:
Why was this changed to 2?

Reply:
@andre-meyer because we are showing only 2 decimals for `share.balance` in the shares table, but initial value for input is 4 decimals

Reply:
So it was like `5.50` in table, but the initial value for input was, for example, `5.5064`

Reply:
Ok, got it